[Site Request] OFTV

### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2022.09.01** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

United States

### Example URLs

- single video: https://of.tv/video/627d7d95b353db0001dadd1a/
- creator: https://of.tv/creators/this-is-fire/

### Provide a description that is worded well enough to be understood

yt-dlp downloads the video (via generic Zype) but ends up going through 6 failures first (generic html5). It thinks the single video link is a playlist. Also doesn't pull the creator name (artist) out of the page. The download archive ends up being stored as `zype 627d7d95b353db0001dadd1a` instead of something more appropriate.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--config-location', 'C:\\Users\\NAME\\yt-dl.conf', '--ffmpeg-location', 'C:\\Users\\NAME\\ffmpeg\\bin\\ffmpeg.exe', '--download-archive', 'C:\\Users\\NAME\\Desktop\\archive.txt', '-f', 'bv*+ba/b', '--embed-thumbnail', '--downloader', 'C:\\Users\\NAME\\aria2\\64bit\\aria2c.exe', '-o', 'C:\\Users\\NAME\\Desktop\\%(upload_date>%Y-%m-%d|)s %(uploader)s - %(title)s_%(id)s.%(ext)s', 'https://of.tv/video/627d7d95b353db0001dadd1a/', '-vU']
[debug] | Config "C:\Users\NAME\yt-dl.conf": ['--no-playlist', '-i', '-w', '--sleep-interval', '7', '--max-sleep-interval', '20', '--embed-metadata', '--no-embed-info-json', '--ppa', 'ThumbnailsConvertor:-q:v 1', '--downloader-args', 'aria2c:--file-allocation=none -x 1 -k 1M']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.09.01 [5d7c7d6] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] Checking exe version: "C:\Users\NAME\ffmpeg\bin\ffmpeg.exe" -bsfs
[debug] Checking exe version: "C:\Users\NAME\ffmpeg\bin\ffprobe.exe" -bsfs
[debug] exe versions: ffmpeg git-2019-10-31-b414cff, ffprobe git-2019-10-31-b414cff
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1670 extractors
[debug] Loading archive file 'C:\\Users\\NAME\\Desktop\\archive.txt'
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.09.01, Current version: 2022.09.01
yt-dlp is up to date (2022.09.01)
[debug] [generic] Extracting URL: https://of.tv/video/627d7d95b353db0001dadd1a/
[generic] 627d7d95b353db0001dadd1a: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] 627d7d95b353db0001dadd1a: Extracting information
[debug] Looking for Brightcove embeds
[debug] Looking for embeds
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Identified 6 html5 embeds
[debug] Identified a Zype embed
[download] Downloading playlist: E1: Jacky vs Eric
[debug] Skipping writing playlist thumbnail
[generic] Playlist E1: Jacky vs Eric: Downloading 7 videos of 7
[download] Downloading video 1 of 7
[info] 627d7d95b353db0001dadd1a-1: Downloading 1 format(s): 0
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"
[download] Sleeping 10.71 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (1)_627d7d95b353db0001dadd1a-1.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (1)_627d7d95b353db0001dadd1a-1.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"

09/24 03:08:27 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.url%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.url
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 2 of 7
[info] 627d7d95b353db0001dadd1a-2: Downloading 1 format(s): 0
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"
[download] Sleeping 15.63 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (2)_627d7d95b353db0001dadd1a-2.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (2)_627d7d95b353db0001dadd1a-2.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"

09/24 03:08:44 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.url%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.url
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 3 of 7
[info] 627d7d95b353db0001dadd1a-3: Downloading 1 format(s): 0
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"
[download] Sleeping 9.88 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (3)_627d7d95b353db0001dadd1a-3.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (3)_627d7d95b353db0001dadd1a-3.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"

09/24 03:08:54 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.url%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.url
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 4 of 7
[info] 627d7d95b353db0001dadd1a-4: Downloading 1 format(s): 0
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"
[download] Sleeping 13.82 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (4)_627d7d95b353db0001dadd1a-4.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (4)_627d7d95b353db0001dadd1a-4.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.url }}"

09/24 03:09:08 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.url%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.url
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 5 of 7
[info] 627d7d95b353db0001dadd1a-5: Downloading 1 format(s): 5
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.model.wav }}"
[download] Sleeping 9.41 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (5)_627d7d95b353db0001dadd1a-5.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (5)_627d7d95b353db0001dadd1a-5.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.model.wav }}"

09/24 03:09:18 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.model.wav%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.model.wav
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 6 of 7
[info] 627d7d95b353db0001dadd1a-6: Downloading 1 format(s): 7
[debug] Invoking aria2c downloader on "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.model.flv }}"
[download] Sleeping 12.50 seconds ...
[download] Destination: C:\Users\NAME\Desktop\ NA - 627d7d95b353db0001dadd1a (6)_627d7d95b353db0001dadd1a-6.unknown_video
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 --min-split-size 1M --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" --header "Referer: https://of.tv/video/627d7d95b353db0001dadd1a/" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" --out ".\ NA - 627d7d95b353db0001dadd1a (6)_627d7d95b353db0001dadd1a-6.unknown_video.part" "--auto-file-renaming=false" -- "https://of.tv/video/627d7d95b353db0001dadd1a/{{ data.model.flv }}"

09/24 03:09:32 [ERROR] CUID#7 - Download aborted. URI=https://of.tv/video/627d7d95b353db0001dadd1a/{{%20data.model.flv%20}}
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://of.tv/video/627d7d95b353db0001dadd1a/%20data.model.flv
  -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found

aria2 will resume download if the transfer is restarted.
If there are any errors, then see the log file. See '-l' option in help/man page for details.


ERROR: aria2c exited with code 3
  File "yt_dlp\__main__.py", line 18, in <module>
  File "yt_dlp\__init__.py", line 962, in main
  File "yt_dlp\__init__.py", line 954, in _real_main
  File "yt_dlp\YoutubeDL.py", line 3293, in download
  File "yt_dlp\YoutubeDL.py", line 3268, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1448, in extract_info
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1556, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1684, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 1829, in __process_playlist
  File "yt_dlp\YoutubeDL.py", line 1459, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1855, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.py", line 1614, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2712, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 3171, in process_info
  File "yt_dlp\YoutubeDL.py", line 2895, in dl
  File "yt_dlp\downloader\common.py", line 443, in download
  File "yt_dlp\downloader\external.py", line 73, in real_download
  File "yt_dlp\YoutubeDL.py", line 994, in report_error
  File "yt_dlp\YoutubeDL.py", line 923, in trouble


[download] Downloading video 7 of 7
[debug] [Zype] Extracting URL: https://player.zype.com/embed/627d7d95b353db0001dadd1a.js?api_key=Y0wc_tjz74PGyf-PULVocVHU4bjAcil5Da_Zb4qTSZSbjDu9uAoLLhbo17cG4VBL&autoplay=true&playlist_id=627d36e8a1e93b00010dc39e
[Zype] 627d7d95b353db0001dadd1a: Downloading JSON metadata
[Zype] 627d7d95b353db0001dadd1a: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] 627d7d95b353db0001dadd1a: Downloading 1 format(s): hls-18022
[info] Downloading video thumbnail 3 ...
[info] Writing video thumbnail 3 to: C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.jpg
[debug] Invoking hlsnative downloader on "https://mf.zype.com/eyJhbGciOiJIUzI1NiJ9.eyJ1cmwiOiJodHRwczovL2d2c20uenlwZS5jb20vNWVjNmM2YWY1ZmNhNjgwMDAxYjVlY2YyLzYyN2Q3ZDk1YjM1M2RiMDAwMWRhZGQxYS82MjdkN2Q5NWIzNTNkYjAwMDFkYWRkMWIvNWRkNThiZTJiMjU0MGU1ZGU1ZDZmM2Q3L2UxZWIyNGMyLTY5OGYtNDA4ZS1hY2MwLWRhNjk0ODIzYmZjOC5tM3U4IiwicGFyYW1zIjp7IjZwTEtNUTN5IjoiU3FpM05nOTAiLCJGMzhiMWRGNCI6IjYzMmViYjNjYmViYzE3MDAwMTlhNDZjOCJ9LCJleHAiOjE2NjQwMTc3NzJ9.ZL8bg9H2qFZAfWRd7R3dsCXc9jBjQ98McfS0OvlGRHY"
[download] Sleeping 18.42 seconds ...
[hlsnative] Downloading m3u8 manifest
[hlsnative] Fragment downloads will be delegated to aria2c
[hlsnative] Total fragments: 143
[download] Destination: C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4
[debug] File locking is not supported. Proceeding without locking
[debug] aria2c.exe command line: "C:\Users\NAME\aria2\64bit\aria2c.exe" -c "--console-log-level=warn" "--summary-interval=0" "--download-result=hide" "--http-accept-gzip=true" "--file-allocation=none" -x16 -j16 -s16 "--allow-overwrite=true" "--allow-piece-length-change=true" --header "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36" --header "Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" --header "Accept-Language: en-us,en;q=0.5" --header "Sec-Fetch-Mode: navigate" "--check-certificate=true" "--remote-time=true" "--show-console-readout=true" "--file-allocation=none" -x 1 -k 1M --dir "C:\Users\NAME\Desktop\" "--auto-file-renaming=false" "--file-allocation=none" "--uri-selector=inorder" -i "C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4.part.frag.urls"
[aria2c] Downloaded 1872105880 bytes [#42ec49 7.1MiB/10MiB(66%)][#95b258 10MiB/11MiB(84%)][#105391 5.7MiB/10MiB(53%)][
[download] 100% of 1.74GiB in 01:02 at 28.39MiB/s
[debug] ffprobe command line: "C:\Users\NAME\ffmpeg\bin\ffprobe.exe" -hide_banner -show_format -show_streams -print_format json "file:C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4"
[FixupM3u8] Fixing MPEG-TS in MP4 container of "C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4"
[debug] ffmpeg command line: "C:\Users\NAME\ffmpeg\bin\ffmpeg.exe" -y -loglevel "repeat+info" -i "file:C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4" -map 0 -dn -ignore_unknown -c copy -f mp4 "-bsf:a" aac_adtstoasc -movflags "+faststart" "file:C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.temp.mp4"
[Metadata] Adding metadata to "C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4"
[debug] ffmpeg command line: "C:\Users\NAME\ffmpeg\bin\ffmpeg.exe" -y -loglevel "repeat+info" -i "file:C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4" -map 0 -dn -ignore_unknown -c copy -write_id3v1 1 -metadata "title=E1: Jacky vs Eric" -metadata "date=20220512" -metadata "description=Singer Jacky Romero and actor Eric Guilmette must step out of their comfort zones in order to take on the ultimate comfort food: the Monte Cristo sandwich. With only 30 minutes to deliver the perfect blend of sweet and savory, Jacky and Eric need all the help they can get to impress Chef JoJo and move on to the next round. 

Subscribe to the cast on OF:
of.com/jackyromero
of.com/eric_g
of.com/grandmasterchefjojo" -metadata "synopsis=Singer Jacky Romero and actor Eric Guilmette must step out of their comfort zones in order to take on the ultimate comfort food: the Monte Cristo sandwich. With only 30 minutes to deliver the perfect blend of sweet and savory, Jacky and Eric need all the help they can get to impress Chef JoJo and move on to the next round. 

Subscribe to the cast on OF:
of.com/jackyromero
of.com/eric_g
of.com/grandmasterchefjojo" -metadata "purl=https://player.zype.com/embed/627d7d95b353db0001dadd1a.js?api_key=Y0wc_tjz74PGyf-PULVocVHU4bjAcil5Da_Zb4qTSZSbjDu9uAoLLhbo17cG4VBL&autoplay=true&playlist_id=627d36e8a1e93b00010dc39e" -metadata "comment=https://player.zype.com/embed/627d7d95b353db0001dadd1a.js?api_key=Y0wc_tjz74PGyf-PULVocVHU4bjAcil5Da_Zb4qTSZSbjDu9uAoLLhbo17cG4VBL&autoplay=true&playlist_id=627d36e8a1e93b00010dc39e" -movflags "+faststart" "file:C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.temp.mp4"
[EmbedThumbnail] mutagen: Adding thumbnail to "C:\Users\NAME\Desktop\2022-05-12 NA - E1： Jacky vs Eric_627d7d95b353db0001dadd1a.mp4"
[debug] Adding to archive: zype 627d7d95b353db0001dadd1a
[download] Finished downloading playlist: E1: Jacky vs Eric
```

--- a/yt_dlp/extractor/_extractors.py
+++ b/yt_dlp/extractor/_extractors.py
@@ -1229,6 +1229,10 @@ from .nzherald import NZHeraldIE
 from .nzz import NZZIE
 from .odatv import OdaTVIE
 from .odnoklassniki import OdnoklassnikiIE
+from .oftv import (
+    OfTVIE,
+    OfTVPlaylistIE
+)
 from .oktoberfesttv import OktoberfestTVIE
 from .olympics import OlympicsReplayIE
 from .on24 import On24IE

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -14,7 +14,7 @@ class OfTVIE(InfoExtractor):
             'title': 'E1: Jacky vs Eric',
             'thumbnail': r're:^https?://.*\.jpg',
             'average_rating': 0,
-            'description': 'Singer Jacky Romero and actor Eric Guilmette must step out of their comfort zones in order to take on the ultimate comfort food: the Monte Cristo sandwich. With only 30 minutes to deliver the perfect blend of sweet and savory,\xa0Jacky and Eric need all the help they can get to impress Chef JoJo and move on to the next round.\xa0\r\n\r\nSubscribe to the cast on OF:\r\nof.com/jackyromero \r\nof.com/eric_g \r\nof.com/grandmasterchefjojo',
+            'description': 'md5:dd16e3e2a8d27d922e7a989f85986853',
             'display_id': '',
             'duration': 1423,
             'timestamp': 1652391300,

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -45,7 +45,7 @@ class OfTVPlaylistIE(InfoExtractor):
         webpage = self._download_webpage(url, playlist_id)
 
         remaining_videos = self._search_json(
-            r'var\s*remaining_videos\s*=', webpage, 'oftv playlists', playlist_id, end_pattern=r';', contains_pattern=r'\[.+?\]')
+            r'var\s*remaining_videos\s*=', webpage, 'oftv playlists', playlist_id, contains_pattern=r'\[.+\]')
 
         return self.playlist_from_matches(
             traverse_obj(remaining_videos, (..., 'discovery_url')), playlist_id)

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -27,7 +27,7 @@ class OfTVIE(InfoExtractor):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
         output = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
-        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>(?P<artist>[^<]+)', webpage, 'creator')
+        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>(?P<creator>[^<]+)', webpage, 'creator')
         return output
 
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -26,7 +26,9 @@ class OfTVIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
-        return next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
+        output = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
+        output['creator'] = self._search_regex(r'<a class=\"creator-name\" href=\".+?\">(?P<artist>[a-zA-Z0-9\s].+)<\/a>', webpage, 'creator')
+        return output
 
 
 class OfTVPlaylistIE(InfoExtractor):

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -27,7 +27,7 @@ class OfTVIE(InfoExtractor):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
         output = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
-        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>(?P<creator>[^<]+)', webpage, 'creator')
+        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>([^<]+)', webpage, 'creator')
         return output
 
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -1,0 +1,48 @@
+from .common import InfoExtractor
+from .zype import ZypeIE
+from ..utils import traverse_obj
+
+
+class OfTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?of.tv/video/(?P<id>[0-9a-zA-Z]+)'
+    _TESTS = [{
+        'url': 'https://of.tv/video/627d7d95b353db0001dadd1a',
+        'md5': 'cb9cd5db3bb9ee0d32bfd7e373d6ef0a',
+        'info_dict': {
+            'id': '627d7d95b353db0001dadd1a',
+            'ext': 'mp4',
+            'title': 'E1: Jacky vs Eric',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'average_rating': 0,
+            'description': 'Singer Jacky Romero and actor Eric Guilmette must step out of their comfort zones in order to take on the ultimate comfort food: the Monte Cristo sandwich. With only 30 minutes to deliver the perfect blend of sweet and savory,\xa0Jacky and Eric need all the help they can get to impress Chef JoJo and move on to the next round.\xa0\r\n\r\nSubscribe to the cast on OF:\r\nof.com/jackyromero \r\nof.com/eric_g \r\nof.com/grandmasterchefjojo',
+            'display_id': '',
+            'duration': 1423,
+            'timestamp': 1652391300,
+            'upload_date': '20220512',
+            'view_count': 0
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        return next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
+
+
+class OfTVPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?of.tv/creators/(?P<id>[a-zA-Z0-9-]+)/.?'
+    _TESTS = [{
+        'url': 'https://of.tv/creators/this-is-fire/',
+        'playlist_count': 8,
+        'info_dict': {
+            'id': 'this-is-fire'
+        }
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+        webpage = self._download_webpage(url, playlist_id)
+        playlists_match = self._search_regex(r'var\s*remaining_videos\s*=\s*(\[.+?\])\s*;', webpage, 'oftv playlists')
+        remaining_videos = self._parse_json(playlists_match, playlist_id)
+        filtered = traverse_obj(remaining_videos, (..., "discovery_url", ))
+        return self.playlist_from_matches(filtered, playlist_id)

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -19,8 +19,7 @@ class OfTVIE(InfoExtractor):
             'duration': 1423,
             'timestamp': 1652391300,
             'upload_date': '20220512',
-            'view_count': 0,
-            'md5': None
+            'view_count': 0
         }
     }]
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -43,8 +43,9 @@ class OfTVPlaylistIE(InfoExtractor):
     def _real_extract(self, url):
         playlist_id = self._match_id(url)
         webpage = self._download_webpage(url, playlist_id)
-        
+
         remaining_videos = self._search_json(
             r'var\s*remaining_videos\s*=', webpage, 'oftv playlists', playlist_id, end_pattern=r';', contains_pattern=r'\[.+?\]')
-        filtered = traverse_obj(remaining_videos, (..., "discovery_url", ))
-        return self.playlist_from_matches(filtered, playlist_id)
+
+        return self.playlist_from_matches(
+            traverse_obj(remaining_videos, (..., 'discovery_url')), playlist_id)

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -19,7 +19,8 @@ class OfTVIE(InfoExtractor):
             'duration': 1423,
             'timestamp': 1652391300,
             'upload_date': '20220512',
-            'view_count': 0
+            'view_count': 0,
+            'md5': None
         }
     }]
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -27,7 +27,7 @@ class OfTVIE(InfoExtractor):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
         output = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
-        output['creator'] = self._search_regex(r'<a class=\"creator-name\" href=\".+?\">(?P<artist>[a-zA-Z0-9\s].+)<\/a>', webpage, 'creator')
+        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>(?P<artist>[^<]+)', webpage, 'creator')
         return output
 
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -42,7 +42,8 @@ class OfTVPlaylistIE(InfoExtractor):
     def _real_extract(self, url):
         playlist_id = self._match_id(url)
         webpage = self._download_webpage(url, playlist_id)
-        playlists_match = self._search_regex(r'var\s*remaining_videos\s*=\s*(\[.+?\])\s*;', webpage, 'oftv playlists')
-        remaining_videos = self._parse_json(playlists_match, playlist_id)
+        
+        remaining_videos = self._search_json(
+            r'var\s*remaining_videos\s*=', webpage, 'oftv playlists', playlist_id, end_pattern=r';', contains_pattern=r'\[.+?\]')
         filtered = traverse_obj(remaining_videos, (..., "discovery_url", ))
         return self.playlist_from_matches(filtered, playlist_id)

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -4,7 +4,7 @@ from ..utils import traverse_obj
 
 
 class OfTVIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?of.tv/video/(?P<id>[0-9a-zA-Z]+)'
+    _VALID_URL = r'https?://(?:www\.)?of.tv/video/(?P<id>\w+)'
     _TESTS = [{
         'url': 'https://of.tv/video/627d7d95b353db0001dadd1a',
         'md5': 'cb9cd5db3bb9ee0d32bfd7e373d6ef0a',

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -19,7 +19,8 @@ class OfTVIE(InfoExtractor):
             'duration': 1423,
             'timestamp': 1652391300,
             'upload_date': '20220512',
-            'view_count': 0
+            'view_count': 0,
+            'creator': 'This is Fire'
         }
     }]
 

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -26,9 +26,9 @@ class OfTVIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
-        output = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
-        output['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>([^<]+)', webpage, 'creator')
-        return output
+        info = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
+        info['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>([^<]+)', webpage, 'creator')
+        return info
 
 
 class OfTVPlaylistIE(InfoExtractor):

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -45,8 +45,8 @@ class OfTVPlaylistIE(InfoExtractor):
         playlist_id = self._match_id(url)
         webpage = self._download_webpage(url, playlist_id)
 
-        remaining_videos = self._search_json(
+        json_match = self._search_json(
             r'var\s*remaining_videos\s*=', webpage, 'oftv playlists', playlist_id, contains_pattern=r'\[.+\]')
 
         return self.playlist_from_matches(
-            traverse_obj(remaining_videos, (..., 'discovery_url')), playlist_id)
+            traverse_obj(json_match, (..., 'discovery_url')), playlist_id)

--- a/yt_dlp/extractor/oftv.py
+++ b/yt_dlp/extractor/oftv.py
@@ -27,6 +27,7 @@ class OfTVIE(InfoExtractor):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
         info = next(ZypeIE.extract_from_webpage(self._downloader, url, webpage))
+        info['_type'] = 'url_transparent'
         info['creator'] = self._search_regex(r'<a[^>]+class=\"creator-name\"[^>]+>([^<]+)', webpage, 'creator')
         return info
 

